styles/globals.css

```diff
@@ -26,10 +26,20 @@ body {
     rgb(var(--background-start-rgb));
 }
 
+progress[value]::-webkit-progress-bar {
```

Comment:
Loving the effort put into accessibility so far tbh. You doing the lord's work man🥂✨

But I feel these can still somehow be made into utility classes or better still utility plugins at worst instead of resorting to @apply, no?🤔

Reply:
no ko, yes ni